escapades_frontend/src/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,27 @@
+.Navbar {
+  @apply sticky top-0 text-white bg-gray-800 flex justify-between items-center gap-8 pl-4 z-[9999] h-8
+          sm:h-16;
+
+  &__link {
+    @apply fixed left-[66%] top-[-100%] flex-col gap-4 p-4 text-sm w-[100%] bg-gray-800
+            sm:static sm:flex sm:left-auto sm:top-auto sm:flex-row sm:text-base sm:w-auto;
+    transition: 300ms;
+  }
+
+  &__linkActive {
+    @apply bg-white text-gray-800;
+  }
```

Comment:
here I wanted to use a BEM modifier like `Navbar__link--active` but typescript doesnt allow this syntax..any better idea ?

Reply:
Have you tried this pattern:

```scss
// Example Module Name: myModule

.myModule {
  // Module styles

  &__block {
    // Block styles

    &--modifier {
      // Modifier styles
    }

    &__element {
      // Element styles

      &--modifier {
        // Modifier styles
      }
    }
  }

  &__anotherBlock {
    // Another block styles
  }
}
```

Reply:
thks but when I try to call the modifier classes in my component ex: `classes.Navbar__links--open` it doesnt work and I have a parsing error :/